bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-hotgraphic",
-  "version": "2.0.7",
+  "version": "2.0.8",
```

Comment:
version should actually be 2.0.9 - some idiot (me!) forgot to bump the version when releasing [2.0.8](https://github.com/adaptlearning/adapt-contrib-hotgraphic/releases/tag/v2.0.8)